californium-core/src/main/java/org/eclipse/californium/core/network/InMemoryMessageExchangeStore.java

```diff
@@ -358,6 +381,52 @@ public synchronized void stop() {
 		}
 	}
 
+	/**
+	 * Dump exchanges of stores.
+	 * 
+	 * @param logMaxExchanges maximum number of exchanges to include in dump.
+	 */
+	public void dump(int logMaxExchanges) {
+		if (LOGGER.isInfoEnabled()) {
```

Comment:
Who will be calling this method and in what context?

Reply:
The health status.
And unit tests.